src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -62,8 +64,15 @@ const SetAclPermsDocContainerForm = () => {
         state,
         dispatch
       );
+      addNotification(
+        'info',
```

Comment:
Similar comment to the other permission form about using 'success'

Reply:
Adding onto this. I was also confused - but that's due to the code that was already there.   

So the fix for this is yes to make it `'success'` but also change the words to `Gave` and `Revoked` - I didn't initially realize this is after it is successfully completed.

Reply:
The more I think about it... this line: 
`${permissions.read ? 'Give' : 'Revoke'} permission to ${permissionState.podUrlToSetPermissionsTo} for Documents Container.`
        
Should be:
`${permissions.read ? 'Gave' : 'Revoked'} permission ${permissions.read ? 'to' : 'from'} ${permissionState.podUrlToSetPermissionsTo} for Documents Container.`